docs/upgrades.md

```diff
@@ -127,9 +127,16 @@ now be ignored.
 
 The default for the helm configuration `service.create` was changed to `true` as
 well. Again, most users have this set to `true` in which case it does not impact
-you. If you were relying on the default to be `false` so you could create your
-own Kubernetes service object for Sematic, you now must explicitly set
-`service.create` to `false` in your helm values.
+you.
+
+If you are using a non-dedicated SocketIO deployment (`deployment.socket_io.dedicated`
+set to `false`, or unspecified), and you want to keep the same setup, you must now
```

Comment:
I think we should dis-allow non-dedicated socketio. That was actually the intent of https://github.com/sematic-ai/sematic/pull/1060

Reason:
- the only people we have heard of who have tried such a setup haven't got it working
- the people we are aware of who have helm deployments all use a dedicated socketio server
- it is not trivial to test & confirm that such a setup continues to work

If anybody complains after we remove support, they can reach out and discuss their use case and only invest in a proper testing setup once we know it's actually worthwhile.

Reply:
Updated.